aspnetcore/web-api/index.md

```diff
@@ -401,9 +355,10 @@ The automatic creation of a `ProblemDetails` instance is disabled when the <xref
 
 ::: moniker-end
 
-## Additional resources 
+## Additional resources
 
 * <xref:web-api/action-return-types>
+* <xref:web-api/error-handling>
```

Comment:
```suggestion
* <xref:web-api/handle-errors>
```

Reply:
Why do you want to change the link and then change the file name? It's not worth the effort.